src/world/BallPossession.cpp

```diff
@@ -69,19 +70,18 @@ bool BallPossession::teamCloseToBall(const world::WorldData &world, bool ourTeam
     return false;
 }
 
-/// return true if given team is relatively far from ball
 bool BallPossession::teamFarFromBall(const world::WorldData &world, bool ourTeam) {
     if (world.ball) {
-        double farThreshHoldDist = 0.4;
+        double farThreshHoldDist = STANDARD_FAR_THRESHOLD;
 
-        if (!ourTeam) {
-            // if the ball is on our side, go more defensive.
-            farThreshHoldDist = world.ball->pos.x < 0.0 ? 0.9 : 0.4;
+        if (!ourTeam && world.ball->getPos().x < this->MIDDLE_LINE_X) {
```

Comment:
Personally I think having a constant for 0 is a bit overkill, but it does no harm.

Reply:
In this case zero actually means something. What if somebody wants sometime to change the reference system to start from the corner of the field instead of the center? I would even go one step further and put this into the constants, while also looking for the places where it is used.

Reply:
Whilst I don´t mind the variable for reading clarity, I think the argument you give is weak. There is virtually no reason for ever changing the reference system within our AI (which has some nice unique properties, like being symmetric around zero and being consistent with the rest of SSL software). It would probably be more advised to simply change a different reference system back to the one our AI accepts, if we need to use one at some point.